packages/internal/src/babel/api.ts

```diff
@@ -0,0 +1,83 @@
+import fs from 'fs'
+import path from 'path'
+
+import type { TransformOptions, PluginItem } from '@babel/core'
+import babelRequireHook from '@babel/register'
+
+import { getPaths } from '../paths'
+
+export const getApiSideBabelPlugins = () => {
+  const rwjsPaths = getPaths()
+  // Plugin shape: [ ["Target", "Options", "name"] ],
+  // a custom "name" is supplied so that user's do not accidently overwrite
+  // Redwood's own plugins.
+  const plugins: TransformOptions['plugins'] = [
+    ['@babel/plugin-transform-typescript', undefined, 'rwjs-babel-typescript'],
+    [
+      require('@redwoodjs/core/dist/babelPlugins/babel-plugin-redwood-src-alias')
+        .default,
```

Comment:
Here too, `.default` because I think.... with the datamigrate hook was doing the transform for us (accidentally)